resources/translations.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+let LocaleRegex = {};
+let LocaleNetRegex = {};
+
+// TODO: figure out some non-var way to require files for node, but not break html includes.
+/* eslint-disable no-var */
+var Regexes;
+var NetRegexes;
+/* eslint-enable */
```

Comment:
Also, I still don't know how to fix this.  When you have files that you want to include when running in node, you need to `require` them.  When you include them in html, some other file will define it.  I don't see a way to handle both without `var`, but maybe there's some more obvious solution that I haven't been able to find while searching.